lib/bundler/installer.rb

```diff
@@ -163,10 +168,14 @@ def generate_standalone_bundler_executable_stubs(spec)
     # that said, it's a rare situation (other than rake), and parallel
     # installation is SO MUCH FASTER. so we let people opt in.
     def install(options)
-      load_plugins
       force = options["force"]
-      jobs = 1
-      jobs = [Bundler.settings[:jobs].to_i - 1, 1].max if can_install_in_parallel?
+      jobs = options.delete(:jobs) do
+        if can_install_in_parallel?
+          [Bundler.settings[:jobs].to_i - 1, 1].max
```

Comment:
Any particular reason why you're removing methods like this?  This example from `Installer.max_threads`?